code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -303,7 +303,7 @@
 			enthrall_safe = 1
 			break
 	if(!C)
-		log_to_dd("something bad happened on enthralling a mob, attacker is [user] [user.key] \ref[user]")
+		log_runtime(EXCEPTION("something bad happened on enthralling a mob, attacker is [user] [user.key] \ref[user]"), user)
```

Comment:
No reason to include the user `\ref` here, especially since the user ends up linked in the error viewer. I'm not even sure if there's a convenient way to VV by ref.